### ISSUE TITLE
3.1 [to update] « Nous lancerons un grand Plan d’investissement de 50 milliards d’euros.»

### ISSUE THREAD

**Issue by LVAI** [Insoumis/programme-macron#80]
## 3.1 « Nous lancerons un grand Plan d’investissement de 50 milliards d’euros.»
Ce plan sur 5 ans sera mis au service des qualifications de tous les citoyens, de la transition écologique, de la révolution numérique, de la modernisation des services publics et de la rénovation urbaine.

## OUI, MAIS PAS COMME ÇA
La stratégie de la politique de l'offre déflationniste, reposant sur une forte réduction des dépenses publiques, associée à un faible investissement, est un désastre : elle fait de la zone euro, pourtant deuxième puissance économique mondiale, une zone de faible croissance et de chômage élevé. Désormais, même le FMI et l'OCDE plaident pour une relance de la dépense publique et de l'investissement. Il est donc urgent de mener une politique d'investissement public, mais la stratégie d'Emmanuel Macron, malgré les apparences, ne va vraiment pas dans cette direction :
La somme de 50 milliards semble insuffisante. Elle permettrait à peine ce qui nécessiterait la transition écologique sur le quinquennat, ce qui ne laisse rien pour le reste.
D'où viendra l'argent ? Emprunter auprès d'agents privés, alors que le taux directeur de la BCE est de 0%, augmente inutilement le poids de la dette, tout en faisant "grossir" la bulle financière. Il faudrait emprunter directement auprès de la banque centrale...
Parallèlement, Emmanuel Macron propose de diminuer les dépenses publiques de 60 milliards en 5 ans, avec donc un bilan de dépense publique de -10 milliards, ce qui reste dans la logique austéritaire, contraire donc aux recommandations du FMI et de l'OCDE.

## Qu'en dit l'«Avenir en Commun» ?

- Engager un plan de relance de l'activité et de l'emploi au service de la transition écologique
- Injecter 100 milliards d'euros supplémentaires dans l'économie pour des investissements écologiquement et socialement utiles

- Transformer les 41 milliards d'euros annuels du pacte de responsabilité et du crédit d'impôt compétitivité (soit 2 % du PIB chaque année, 200 milliards d'euros sur un quinquennat) pour financer la transition énergétique et les activités socialement utiles

- Remettre à plat l'ensemble des aides publiques et exonérations d'impôt ou de cotisation accordées aux entreprises, évaluer leur efficacité, et revenir sur les aides antisociales et anti-écologiques

- Réindustrialiser le pays par l'investissement, le rétablissement de plans de filières pour coordonner donneurs d'ordres et sous-traitants, clients et fournisseurs (par exemple SNCF et Alstom pour la filière ferroviaire, etc.), la création de pôles publics dans l'énergie ou les transports, la défense des industries stratégiques et la reconstruction de conglomérats combinant plusieurs activités

- Plan A. Proposer une refondation démocratique, sociale et écologique des traités européens par la renégociation
- Mettre fin à l'indépendance de la Banque centrale européenne, modifier ses missions et status, autoriser le rachat de la dette publique directement aux États, interdire à la BCE de couper les liquidités à un État membre. Sans attendre, la Banque de France sera mise au service de ces objectifs

- Dévaluer l'euro pour revenir à la parité initiale avec le dollar

- Mettre au pas la finance, prohiber les instruments financiers toxiques, taxer les transactions financières, contrôler les mouvements de capitaux pour empêcher les attaques spéculatives

- Organiser une conférence européenne sur les dettes souveraines débouchant sur des moratoires, une baisse des taux d'intérêt, des rééchelonnements et annulations partielles

- Arrêter la libéralisation des service publics (rail, énergie, télécoms…)

- Mettre en place un protectionnisme solidaire : arrêt de la libre circulation des capitaux et des marchandises entre l'UE et les pays tiers, arrêt des politiques de libre-échange qui ruinent les économies en développement et détruisent l'industrie européenne, autorisation des aides d'État aux secteurs stratégiques

- Mettre fin au dumping à l'intérieur de l'UE par une politique volontariste et rapide d'harmonisation sociale et fiscale par le haut dans toute l'UE avec inscription d'une clause de non-régression des droits sociaux

- Refonder la politique agricole commune pour garantir l'autosuffisance alimentaire, la relocalisation et l'agriculture écologique et paysanne

- Abandonner le marché carbone et mettre en œuvre une véritable politique de réduction des émissions de gaz à effet de serre avec des critères de convergence impératifs

## Contribution requise 

Ajouter une critique plus détaillées sur les 50 milliards d'investissement (notamment la part sur la transition éco qui est une grosse plaisanterie).

Clarifier les -10 milliards. D'un côté on parle de 50 milliards qui sont de l'investissement et de l'autre il parle de baisse de la dépense publique de 60 milliards donc on dit - 10 milliards. Mais on ne mélange pas de l'investissement avec des flux de dépenses.

**Comment by Scalpi** [comment]
Voire suspect ou démagogique. Il ne peut pas proposer un tel plan soi-disant ambitieux avec 50 milliards. Nous avons étudié son programme sur la transition écologique. Il prévoirait notamment de fermer 50% du parc nucléaire. Sur la transition en général, il consacre 15 milliards sur les 50. Est-ce sérieux de vouloir fermer les 5 centrales thermiques à charbon en 5 ans et de vouloir fermer la moitié du parc nucléaire qui suppose de convertir tout ceci en énergies renouvelables avec 15 milliards à horizon 2025 ?

**Comment by lucasgautheron** [comment]
@LVAI pourquoi ce ticket ? qu'est-ce qui a changé par rapport à la version actuelle ?

**Comment by lucasgautheron** [comment]
"Clarifier les -10 milliards. D'un côté on parle de 50 milliards qui sont de l'investissement et de l'autre il parle de baisse de la dépense publique de 60 milliards donc on dit - 10 milliards. Mais on ne mélange pas de l'investissement avec des flux de dépenses." > j'ai fait ça sous la caution de Jacques Généreux :D je l'ai entendu lui même utiliser cet argument.

**Comment by LVAI** [comment]
Comme vu sur Discord (j'avais pas vu ton tag, sorry), rien n'a changé. Il y a juste une proposition d'amélioration ("contribution requise").

**Comment by Scalpi** [comment]
Oui, j'ai entendu aussi l'argument de JG mais je trouve que ce n'est pas
rigoureux de comparer de l'investissement à des économies de dépenses
(comme si tu compares OPEX et CAPEX). L'économie de 60 milliards ne remet
pas en cause le fait qu'il y aura un investissement de 50 milliards. Je
pense que Jacques a voulu user d'une tactique face aux journalistes qui ne
comprennent rien. Mais bon, on peut laisser passer nous aussi, up to you

2017-04-04 11:59 GMT+02:00 LVAI <notifications@github.com>:

**Comment by lucasgautheron** [comment]
Pour moi oui, d'ailleurs au final, quand tu retires 60 milliards de la dépense publique, il faut se demander sur quoi vont se faire les économies. Souvent les investissements en pâtissent beaucoup, puis la fonction publique, bref chute de l'activité et de la consommation et donc contraction. Et l'argent va où ? vu le programme de Macron, il va pas soutenir l'"économie réelle" (c'est schématique mais voilà l'idée)
